### PR TITLE
fix(SUP-38407): [GaTech] [Multistream - V7 Player] Child entries not loading high quality flavor assets

### PR DESCRIPTION
**Pull request by Tzipi-kaltura**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

export changeChildQuality function

part of - https://github.com/kaltura/playkit-js/pull/793/files, https://github.com/kaltura/playkit-js-dual-screen/pull/155

#### Resolves [SUP-38407](https://kaltura.atlassian.net/browse/SUP-38407)

[SUP-38407]: https://kaltura.atlassian.net/browse/SUP-38407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]
fix lint issues

**Comment by Tzipi-kaltura** [comment]
> fix lint issues

yes i need to merge first the playkit-js pull request so it will find the function.

File: src/kaltura-player.ts [hunk 1]
```diff
@@ -438,6 +438,10 @@ export class KalturaPlayer extends FakeEventTarget {
     this._localPlayer.showTextTrack();
   }
 
+  public changeChildQuality(track: any | string): void {
```
Comment (MosheMaorKaltura) [inline src/kaltura-player.ts:441]: rename to changeQuality

Comment (Tzipi-kaltura) [reply src/kaltura-player.ts]: done

**Review by MosheMaorKaltura — APPROVED** [review]

**Review by MosheMaorKaltura — APPROVED** [review]

**Pull request merged**